src/marqo/core/models/marqo_add_documents_response.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Optional, Any
+
+from pydantic import Field, root_validator
+
+from marqo.base_model import MarqoBaseModel
+
+
+class MarqoAddDocumentsItem(MarqoBaseModel):
+    """A response from adding a document to Marqo.
+
+    This model takes the response from Marqo vector store and translate it to a user-friendly response.
+    """
+    status: int
+    # This id can be any type as it might be used to hold an invalid id response
+    id: Any = Field(alias="_id", default=None)
+    message: Optional[str] = None
+    error: Optional[str] = None
+    code: Optional[str] = None
+
+
+class MarqoAddDocumentsResponse(MarqoBaseModel):
+    errors: bool
+    processingTimeMs: float
+    index_name: str  # TODO Change this to camelCase in the future (Breaking change!)
+    items: List[MarqoAddDocumentsItem]
+    success_count: int = Field(exclude=True, default=0)
+    error_count: int = Field(exclude=True, default=0)
+    failure_count: int = Field(exclude=True, default=0)
```

Comment:
Since this is used in 3 different places, is it worth extracting them to a new Model class? `BatchResponseCount` or `BatchResponseStats`?

Reply:
Good idea